odoo/fields.py

```diff
@@ -435,10 +435,14 @@ def _setup_attrs(self, model, name):
             attrs['store'] = attrs.get('store', False)
             attrs['copy'] = attrs.get('copy', False)
             attrs['readonly'] = attrs.get('readonly', not attrs.get('inverse'))
+            if attrs.get('default'):
+                _logger.warning("Field %s/%s: computed field using default value", model, name)
```

Comment:
Please use `%s.%s` instead of `%s/%s` in the format string. In the ORM, I always use the convention *model_name*.*field_name* when referring to a field.